packages/manager/modules/billing/src/payment/method/add/views/legacyBillingAddress/index.html

```diff
@@ -1,103 +1,116 @@
 <div class="billing-payment-method-add-legacy-bank-account-owner">
-    <!-- Owner Name -->
-    <oui-field
-        data-label="{{ :: 'billing_payment_method_add_bank_account_owner_name' | translate }}"
-    >
-        <input
-            type="text"
-            class="oui-input"
-            id="ownerName"
-            name="ownerName"
-            data-ng-model="$ctrl.model.billingAddress.ownerName"
-            data-required
-            data-ng-minlength="5"
-            data-ng-maxlength="255"
-        />
-    </oui-field>
-    <!-- End of Owner Name -->
-
-    <div data-ng-if="$ctrl.currentUser.billingCountry === 'FR'">
-        <!-- Address Number -->
-        <oui-field
-            data-label="{{ :: 'billing_payment_method_add_bank_account_address_number' | translate }}"
-        >
-            <input
-                type="text"
-                class="oui-input"
-                id="addressNumber"
-                name="addressNumber"
-                data-ng-model="$ctrl.model.billingAddress.addressNumber"
-                data-ng-maxlength="255"
-            />
-        </oui-field>
-        <!-- End of Address Number -->
-
-        <!-- Address Street -->
+    <div class="col-md-6">
+        <!-- Owner Name -->
         <oui-field
-            data-label="{{ :: 'billing_payment_method_add_bank_account_address_street' | translate }}"
+            data-label="{{ :: 'billing_payment_method_add_bank_account_owner_name' | translate }}"
         >
             <input
                 type="text"
                 class="oui-input"
-                id="addressStreet"
-                name="addressStreet"
-                data-ng-model="$ctrl.model.billingAddress.addressStreet"
+                id="ownerName"
+                name="ownerName"
+                data-ng-model="$ctrl.model.billingAddress.ownerName"
                 data-required
+                data-ng-minlength="5"
                 data-ng-maxlength="255"
             />
         </oui-field>
-        <!-- End of Address Street -->
+        <!-- End of Owner Name -->
 
-        <!-- Address Zip -->
-        <oui-field
-            data-label="{{ :: 'billing_payment_method_add_bank_account_address_zip' | translate }}"
-        >
-            <input
-                type="text"
-                class="oui-input"
-                id="addressZip"
-                name="addressZip"
-                data-ng-model="$ctrl.model.billingAddress.addressZip"
-                data-required
-                data-ng-pattern="/^\d{5}$/"
-                min="01000"
-                max="99999"
-            />
-        </oui-field>
-        <!-- End of Address Zip -->
+        <div data-ng-if="$ctrl.currentUser.billingCountry === 'FR'">
```

Comment:
Nitpick: I see that the existing code is directly using subsidiary 'FR'. If you have time, move this logic to feature flipping.